src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -191,6 +198,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $io->comment('Re-run the command with a -vv option to see logs about consumed messages.');
         }
 
+        if ($queues = $input->getOption('queue')) {
+            foreach ($receivers as $receiver) {
+                if (!$receiver instanceof QueueAwareInterface) {
+                    throw new \RuntimeException();
+                }
+                $receiver->setQueues($queues);
```

Comment:
It is not rather, `setQueueNames` ?

Reply:
probably. however, i am waiting for input on the general architecture before tweaking the naming

Reply:
we now managed to find a way using the options rather than changing state in the receiver. i have removed this method